config/tips.txt

```diff
@@ -132,3 +132,73 @@ Changelings can spawn as jobs that might otherwise not be spawn-able as a Traito
+Roleplay is actually the easiest part of this game. If you roleplay badly, at least the only one hurt is you. Not so if you mess up the supermatter!
+As an antagonist, you can Set Ambition with a summary of what you intended/set out on doing. This will show up on the Round End summary.
+You don't have to be to be the perfect objective-completing machine. It makes a character much more believable if you have to work around flaws instead of ignoring them when convenient.
+Runechat (Floating chat messages) can be toggled off in the character setup menu, if you prefer the old-school system.
+You can ban somebody's device from the NTNet using the administration utility program.
+Writing in paper has a character limit of 6,000. NanoWord allows 30,000 characters. Good for writing books!
+Examining things shows a message about it to people in a 4 tile range. If you don't want to see this message, you can disable it with a preference.
+Constructing a barricade and then attaching rods to the barricade will hurt anyone who touches the barricade.
+CPR works - it helps with blood circulation even if you fail to restart the heart.
+Engraved graffiti, papers on notice boards, and sticky notes purchased from cargo will persist from round to round.
+As a dirty detective, you can remove the tape from a tape recorder and use wirecutters to manipulate it.
+Skrell and Tritonians handle water very easily.
+Maintenance drones will not disturb fellow synthetics.
+As an IPC or FBP, there is a verb to change your eye color using RGB at will.
+Lung damage occurs in a severe change in pressure. A space suit prevents this, but you will still need internals to avoid suffocation.
+Using a multitool on a smartgun will disable the tracker, but prevents it from firing. An emag works better...
+The deck management program will allow you to file a flight plan, giving some information on shuttle movements and automatically announcing them across the ship.
+Gas inhalation is reagent-based: treat yourself with dialysis/anti-toxin as your lungs send the phoron particles into your bloodstream.
+Melee weapons have different attack cooldowns. Generally small knifes are faster, big toolboxes and axes are slower.
+If you aren't wearing magboots, you will slip in zero-gravity.
+Accuracy also affects melee and disarms: you are a lot less likely to hit if you are blind, blurry eyes, confused, in pain (hearing audible heart beating sound), etc...
+Grabbing a limb and attacking with help intent will Examine, checking for external injuries, bone fractures, or poorly concealed implants.
+Blood spraying out in shots from a limb is a rupture of a major blood vessel, causing a quick death - delay it by using gauze and applying pressure with your hand or a medical splint.
+Cut a hole in the elevator shaft and throw your enemies down it. Leave them to cringe and whimper in a pool of their own blood and bone fractures.
```

Comment:
eeeeeeeh again, map might not always be the Nerva, so I'm not really too fond of map-specific tips personally